contracts/test/oracle/oracle.fork-test.js

```diff
@@ -54,4 +60,126 @@ describe("ForkTest: Oracle Routers", function () {
       }
     });
   });
+  describe("OETH Oracle", () => {
+    it("Should be initialized", async () => {
+      const { oethOracle, oethOracleUpdater, oethVault } = fixture;
+
+      expect(await oethOracleUpdater.governor()).to.eq(
+        addresses.mainnet.Timelock
+      );
+      expect(await oethOracleUpdater.MAX_VAULT_PRICE()).to.eq(
+        parseUnits("0.995")
+      );
+      expect(await oethOracleUpdater.curvePool()).to.eq(
+        addresses.mainnet.CurveOETHMetaPool
+      );
+      expect(await oethOracleUpdater.vault()).to.eq(oethVault.address);
+
+      expect(await oethOracle.decimals()).to.eq(18);
+      expect(await oethOracle.version()).to.eq(1);
```

Comment:
would be cool to have a test that maps deployed implementation addresses to versions. And when we attempt to update the Oracle and forget to also bump the version that test would fail.